src/main/java/frc/robot/RobotContainer.java

```diff
@@ -93,12 +88,11 @@ public class RobotContainer {
   public static AlgaeClaw algaeClaw;
   public static Climber climber;
   // Xbox Controllers
-  private final CommandXboxController driver = new CommandXboxController(0);
-  private static final CommandXboxController operator = new CommandXboxController(1);
+  private DriverControls driverControls = new DriverControls();
+  private OperatorControls operatorControls = new OperatorControls(driverControls, climber);
   private Trigger reefAlignTrigger = new Trigger(ReefAlign.getInstance()::shouldDoReefAlign);
   private Trigger sourceAlignTrigger = new Trigger(SourceAlign.getInstance()::shouldDoSourceAlign);
   private static boolean hasRanAuto = false;
```

Comment:
These Triggers are events that change how the driver controls the robot. Maybe they would also be better placed in `DriverControls.java`? That would also mean that we wouldn't have a `configureButtonBindings` method at all

Just a suggestion, not a requirement... this is still way cleaner overall!

Reply:
sounds good, ill merge this in now then to get some progress on the refactors into main